packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -50,6 +50,10 @@ export class AtomicSearchBox {
    * Whether the submit button should be placed before the input.
    */
   @Prop() leadingSubmitButton = false;
+  /**
+   * Whether the filters should be cleared upon a new search
```

Comment:
Small detail, should end with a dot.

Another thing I forgot to mention that is important until we don't have a ref doc: `npm run build` has to be run to update the readmes of the component. Those readmes then have to be commited.